git-branchless-lib/src/git/run.rs

```diff
@@ -418,6 +418,11 @@ impl GitRunInfo {
             if let Some(path) = env.get(OsStr::new("PATH")) {
                 path_components.extend(std::env::split_paths(path));
             }
+            // On windows, PATH's name defaults to "Path".
+            #[cfg(target_os = "windows")]
+            if let Some(path) = env.get(OsStr::new("Path")) {
```

Comment:
When possible, I like to use `if !cfg(target_os = "windows")` instead, as the compiler will typecheck the body of the `if`-expression then, but not a big deal here.